srv/components/system/firewall/sanity_check.sls

```diff
@@ -102,7 +102,3 @@ Verify {{ network_interface }} ports:
 
 {% endfor %}
 
```

Comment:
Removed the step from `sanity_check`, as firewall validations command can be individually run as a manual step post-deployment.